src/test/java/io/strimzi/kafka/bridge/http/ConsumerTest.java

```diff
@@ -31,6 +32,7 @@
     private static final String FORWARDED = "Forwarded";
     private static final String X_FORWARDED_HOST = "X-Forwarded-Host";
     private static final String X_FORWARDED_PROTO = "X-Forwarded-Proto";
+    private static long timeout = 5L;
```

Comment:
Can we avoid to have this declared twice? It's in the HttpBridgeTestBase class as well.